shavar/views/__init__.py

```diff
@@ -88,6 +88,15 @@ def format_downloads(request, resp_payload):
 
     for lname, ldata in resp_payload['lists'].iteritems():
         body += "i:%s\n" % lname
+
+        # Not publishing deltas for this list?  Delete all previous chunks to
+        # make way for the new corpus
+        if (_setting(request, lname, 'not_publishing_deltas')
+                and len(ldata['adds']) == 1
+                and len(ldata['subs']) == 0):
```

Comment:
I'd suggest actually erroring out if there's not exactly one add chunk and zero sub chunks, 'cause it means someone needs to fix their config